PolyEngine/Engine/Src/Input/InputWorldComponent.hpp

```diff
@@ -24,7 +24,10 @@ namespace Poly
 	public:
 		RTTI_DECLARE_COMPONENT(::Poly::InputWorldComponent) { NO_RTTI_PROPERTY(); }
 
-		InputWorldComponent() = default;
+		InputWorldComponent()
+			: IsConsumed(false), CharUTF8(nullptr)
```

Comment:
This should have been done in class header, not in the constructor. Please revert this.

Reply:
Done